src/Uno.UI/UI/Xaml/Media/RadialGradientBrush.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Windows.Foundation;
+using Windows.UI.Composition;
+using Windows.UI.Xaml.Markup;
+
+namespace Windows.UI.Xaml.Media
```

Comment:
Should this be the `Microsoft.UI.Xaml.Media` namespace? cc @jeromelaban

Reply:
Indeed:

https://github.com/microsoft/microsoft-ui-xaml/blob/ca95be6f5248eaa55ec523f2d6e7b50f6e0d53e7/dev/RadialGradientBrush/RadialGradientBrush.idl#L1

In WinUI 2.x, this type is used by qualifying explicitly in the xaml.